PiwikTracker/PiwikTracker.swift

```diff
@@ -128,6 +128,14 @@ final public class PiwikTracker: NSObject {
     internal var visitor = Visitor.current()
     internal var session = Session.current()
     internal var nextEventStartsANewSession = true
+    
+    public func userAgent () -> String {
+        return self.dispatcher.userAgent!
+    }
```

Comment:
I don't think force-unwrapping is a good idea here. The userAgent of a Dispatcher is optional. So it could be that it is nil. In this case it would lead to a crash.